rules/identify-sql-performance-sql-server/rule.md

```diff
@@ -0,0 +1,72 @@
+---
+type: rule
+archivedreason: 
+title: Identifying the cause of SQL Server performance problems
+guid: 46f8cae3-7aa0-4313-9ac2-8e9d84ef236c
+uri: identify-sql-performance-sql-server
+created: 2023-12-27T07:26:54.0000000Z
+authors:
+  - title: Bryden Oliver
+    url: https://ssw.com.au/people/bryden-oliver
+related: 
+  - identify-sql-performance-azure
+  - identify-sql-performance-sql-server
+  - sql-server-cpu-pressure
+  - sql-server-io-pressure
+  - sql-server-memory-pressure
+---
+## Identifying CPU Pressure
+
+When looking at SQL Server, you often get performance issues, but how can you figure out what might be the cause?
+
+<!--endintro-->
+
+To figure out whether the SQL Server itself is experiencing CPU pressure, fire up Task manager and take a look at the CPU usage. If the CPU is high and SQL Server is not the primary consumer, evaluate whether you can separate the CPU consuming workload from your SQL Server by moving one onto another server.
+
+A good way to identify if SQL Server is experiencing CPU Pressure internally is to try the following SQL
```

Comment:
**nitpick**: Ending punctuation.
**nitpick**: The below query is complicated and took me a few moments to understand - could a short explanation be added? Something simple like "this query counts the batch requests, compilations and recompilations that occur over the course of a minute."